app/Domain/Auth/Templates/login.blade.php

```diff
@@ -56,7 +58,7 @@
         <?php $tpl->dispatchTplEvent('beforeSubmitButton'); ?>
 
         <div class="">
-            <x-global::forms.button type="submit" content-role="primary" name="login">
+            <x-global::forms.button type="submit" content-role="primary" name="login" cornerShape="full" :takeFullWidth="true">
                 {{ $tpl->language->__('buttons.login') }}
```

Comment:
for the full width attribute can we just use "scale" and add full-width as an option

Reply:
removed cornerShape and takeFullWidth attribute. using class="w-full"